### ISSUE TITLE
KindlingCenter 

### ISSUE THREAD

**Issue by hiimdemil** [PyroPets/pyropets-docs#1]
It would be more of a type of suggestion regarding KindlingCenter since I consider it to be a tool that provides a better experience when playing with your Pyropets.

Since it makes the task easier, it should stand out a little more than the others. As if it were a recommendation for the owners of Pyropets.

Then it would be to give more prominence to this tool on the main site.

**Comment by SeqSEE** [comment]
Yeah I think this could work. Even though it's technically an experimental feature, documenting it in the docs could benefit hodlers.
